2.4/run-mongod.sh

```diff
@@ -14,108 +15,96 @@ export MONGODB_DATADIR=/var/lib/mongodb/data
 export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
 export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
 export MONGODB_QUIET=${MONGODB_QUIET:-true}
-
 export MONGODB_CONFIG_PATH=/var/lib/mongodb/mongodb.conf
-export MONGODB_PID_FILE=var/run/mongodb/mongodb.pid
-
-MAX_ATTEMPTS=60
-SLEEP_TIME=1
+export MONGODB_PID_FILE=/var/lib/mongodb/mongodb.pid
 
 function usage() {
-	echo "You must specify following environment variables:"
-	echo "  \$MONGODB_USERNAME"
```

Comment:
@jhadvig action item for you.... remove the '$' prefix... it is highly unpractical when copy&pasting the variable in terminal ;-)